students/cocokaku/lesson8/test_circle.py

```diff
@@ -0,0 +1,158 @@
+from circle import Circle
+
+
+# STEP 1 TESTS
+def test_circle_init():
+    c = Circle(4)
+    assert c.radius == 4
+
+
+def test_circle_init_no_parameters_error():
+    msg = ""
+    try:
+        Circle()
+    except TypeError as e:
+        msg = e.args[0]
+    finally:
+        assert msg == "__init__() missing 1 required positional argument: 'radius'"
```

Comment:
use `pytest.raises` to verify raised exceptions:
https://docs.pytest.org/en/latest/assert.html#assertions-about-expected-exceptions